CHANGELOG.md

```diff
@@ -1,13 +1,26 @@
 # Changelog
 
-## [Unreleased]
+## [0.17.0]
 
 ### Added
 
 - Added an experimental swagger definition that includes the specification for
   the vsock API call.
 - Added a signal handler for `SIGBUS` and `SIGSEGV` that immediately terminates
   the process upon intercepting the signal.
+- Added documentation for signal handling utilities.
+- Added [alpha] aarch64 support.
+- Added metrics for read and written bytes by the MMDS, Net and Block Devices.
+
+### Changed
+
+- Unmount and mount events propagate into the chroot location of the jailer.
+- Disallow invalid seccomp levels by exiting with error.
+
+### Fixed
+
+- Fixed swagger definition for `machine-config`.
```

Comment:
This entry is not needed because it fixes a commit that was also introduced in this release.

We do need two additional entries in the changelog:
- in `Changed`: `mem_size_mib`, `ht_enabled` and `vcpu_count` are now required fields when doing a PUT request. Also, PUT requests are now resetting all existing fields.
- in `Added`: PATCH for machine-config.

See commit d4c14d82db3eb936739819f0bb615eebbd1a5291

Reply:
> This entry is not needed because it fixes a commit that was also introduced in this release.

I do not understand what you mean with "an entry fixes a commit that was also introduced in this release".
[Later Edit] I think you mean that It was introduced by a commit that fixed a miss of d4c14d8. I removed it.


> We do need two additional entries in the changelog:
> 
> * in `Changed`: `mem_size_mib`, `ht_enabled` and `vcpu_count` are now required fields when doing a PUT request. Also, PUT requests are now resetting all existing fields.
> * in `Added`: PATCH for machine-config.
> 
> See commit [d4c14d8](https://github.com/firecracker-microvm/firecracker/commit/d4c14d82db3eb936739819f0bb615eebbd1a5291)

Done, take a look.